src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -87,22 +88,21 @@ public synchronized CommandResult execute(String commandText)
 
     @Override
     public synchronized void eagerEvaluate(String commandText, Consumer<String> displayResult) {
-        final Command command = addressBookParser.eagerEvaluateCommand(commandText);
```

Comment:
@CarbonGrid, I edited this part for the eager evalution

Reply:
This one is not a bug. It does not make sense to eager evaluate on for additional space.

Reply:
In a sense, we can specify, eager evaluation will not be run if there is no meaningful changes to the command in command box.